psse/psse-converter/src/test/java/com/powsybl/psse/converter/PsseImporterTest.java

```diff
@@ -107,15 +112,36 @@ public void importTestT3W() throws IOException {
         importTest("ThreeMIB_T3W_modified", "ThreeMIB_T3W_modified.RAW", true);
     }
 
-    @Test(expected = PsseException.class)
-    public void badVersionTest() {
+    @Test()
+    public void badModeTest() {
         ReadOnlyDataSource dataSource = new ResourceDataSource("case-flag-not-supported", new ResourceSet("/", "case-flag-not-supported.raw"));
-        new PsseImporter().importData(dataSource, new NetworkFactoryImpl(), null);
+        PsseImporter psseImporter = new PsseImporter();
+        NetworkFactory networkFactory = new NetworkFactoryImpl();
+        PsseException x = assertThrows(PsseException.class, () -> {
+            psseImporter.importData(dataSource, networkFactory, null);
+        });
+        assertEquals("Incremental load of data option (IC = 1) is not supported", x.getMessage());
```

Comment:
I like the way this can be expressed using `assertj`. Even people from `junit` recommend it (https://github.com/junit-team/junit5/issues/2128). And we already have `assertj` in the project, is not a new dependency.